### ISSUE TITLE
Chisel bypasses FTB claims

### ISSUE THREAD

**Issue by TDanks2000** [CoolMineman/BitsAndChisels#178]
1: make a chisel from chisel and bits
2: goto a claimed area you are not a member of
3: use a chisel on their land

using modpack: `BCG+`
mod version: `2.7.3`
mc version: `1.20.1`

**Comment by jmadarg** [comment]
Indeed, the Chisel is completely capable of bypass FTB Chunks claimd. A message from another mod dev we asked for help:

"The reason why the claim mods (FLAN, GOML, FTB) fail to integrate with bits & chisels is because the chiseling interaction logic happens client-side. It is obviously impossible for a server side mod to prevent client code from running. Additionally this poses a massive security concern as a hacked client can send requests to chisel blocks anywhere in the world while bypassing the break tick cooldown. 

A proper solution would be to eliminate client-side logic from the mod and move those relevant calculations on the server instead. With that it should possible to remove "FlanIsStupid" and "GomlIsVeryStupid" and allow other mods to behave properly with item/block interactions."
